index.js

```diff
@@ -186,6 +188,12 @@ HomeAssistantPlatform.prototype = {
           accessory = new HomeAssistantSwitch(that.log, entity, that, 'input_boolean')
         }else if (entity_type == 'fan'){
           accessory = new HomeAssistantFan(that.log, entity, that)
+        }else if (entity_type == 'binary_sensor'){
+          if (entity.attributes && entity.attributes.sensor_class == 'opening') {
+            accessory = new HomeAssistantBinarySensor(that.log, entity, that, Service.Door, Characteristic.CurrentPosition, 100, 0)
```

Comment:
Also, it would be good to allow users to specify whether this is a door or window with a customize parameter (something like `homebridge_opening_binary_sensor_type`). If door, use `Service.Door` and if window use `Service.Window`. Otherwise you're going to get a lot of false positives for those that have sensors on things other than doors.